src/gui/book/Book.cpp

```diff
@@ -64,7 +64,7 @@ class StatsPage : public PlayerBookPage {
 	void RenderBookPlayerCharacter();
 	bool CheckAttributeClick(Vec2f pos, float * val, TextureContainer * tc);
 	bool CheckSkillClick(Vec2f pos, float * val, TextureContainer * tc, float * oldval);
-	Color attrubuteModToColor(float modValue, float baseValue = 0.f);
+	Color attributeModToColor(float modValue, float baseValue = 0.f);
```

Comment:
Forgot to merge this with the earlier commit?

Reply:
This is just a fix of the original spelling, I thought doing the rename in addition to what the commit does was a bit too much for one commit.